src/main.rs

```diff
@@ -15,23 +16,28 @@ const SLEEP_TIME_MS: u64 = 50;
 const MINIDSP_GAIN_MIN: f64 = -127.0;
 const MINIDSP_GAIN_MAX: f64 = 0.0;
 
+// Lazy static needed since Regex::new() is a function call
+lazy_static! {
+    static ref GAIN_REGEX: Regex =
+        Regex::new(r"Gain\((?P<gain>-*\d*\.*\d*)\)").expect("Failed to create regex for gain");
+}
+
 fn get_gain() -> f64 {
     let output = Command::new(MINIDSP_BINARY)
```

Comment:
There seems to be an API that "minidsp" uses, however it would in practice mean re-implementing the whole "minidsp" program... I know that making system calls are horrible, but I don't really see any other solution.